packages/backend-common/package.json

```diff
@@ -82,7 +82,7 @@
     "fs-extra": "10.1.0",
     "git-url-parse": "^13.0.0",
     "helmet": "^6.0.0",
-    "isomorphic-git": "^1.8.0",
+    "isomorphic-git": "^1.21.0",
```

Comment:
Let's bump all the way to 1.23.0 while at it, and bump this in the scaffolder backend as well